File: MiniETicaretClient/src/app/contracts/order/single_order.ts
export class SingleOrder {
  id: string;
  address: string;
  basketItems: BasketItem[];
  createdDate: Date;
  description: string;
  orderCode: string;
  completed: boolean;
}

export class BasketItem {
  productName: string;
  quantity: number;
  unitPrice: number;
}

File: MiniETicaretAPI/Core/MiniETicaretAPI.Application/Dtos/Order/SingleOrder.cs
namespace MiniETicaretAPI.Application.Dtos.Order
{
    public class SingleOrder
    {
        public string Id { get; set; } = null!;
        public string Address { get; set; }  = null!;
        public object BasketItems { get; set; } = null!;
        public DateTime CreatedDate { get; set; }
        public string Description { get; set; } = null!;
        public string OrderCode { get; set; } = null!;  
        public bool Completed { get; set; }
    }
}
